corehq/apps/hqwebapp/management/commands/hqdefine_to_esm.py

```diff
@@ -147,7 +147,7 @@ def parse_one_line_arguments(self, line, index):
         if match:
             self.update_parser_location(line, index)
             arguments = match.group(1)
-            return [arg.strip() for arg in arguments.split(',')]
+            return [(arg.strip(), '') for arg in arguments.split(',')]
```

Comment:
I know this is just an internal tool, but I do find this hard to wrap my head around on first read. Like the general bigger picture of what this function is doing. Maybe an example in the docstring or a comment explaining why to return this tuple could be useful?

Reply:
fair, added a commit with some comments, let me know what you think